### PR TITLE
Readme corrections

### PR DESCRIPTION
**Pull request by mareq**

### CODE REVIEW

**Comment by mareq** [comment]
Ping: Is this project still active? @paolochiodi

File: README.md [hunk 1]
```diff
@@ -108,9 +108,9 @@ The methods return a new or updated `traceData` to be used for further calls
 <a name="annotationData"></a>
 Annotation Data contains additional info about the current method:
 - service: the name of the current service. It should be an identifier of running process (or class of processes), i.e.: `web_server`, `background_worker`, `checkout_process`. This will be displayed in the zipkin console
-- name: the name of the method being tracked, i.e.: `POST /user` or `update_credentials`. Thi will be displayed in the zipking console on the single span
+- name: the name of the method being tracked, i.e.: `POST /user` or `update_credentials`. This will be displayed in the zipking console on the single span
 
-<a name="sendClientReceive"></a>
+<a name="sendClientRecv"></a>
```
Comment (paolochiodi) [inline README.md:113]: not sure to understand why this change

**Comment by paolochiodi** [comment]
@mareq I'm not actively using this anymore, but I'm happy to spend some time here and there if necessary

File: README.md (hunk 1 above)
Comment (paolochiodi) [reply README.md:113]: ah, wait, I got it. I think you should also change the following line

**Comment by paolochiodi** [comment]
Also, thanks for taking the time to submit the PR
